docs/api/combineActions.md

```diff
@@ -51,7 +51,7 @@ const { increment, decrement } = createActions({
 });
 
 const reducer = handleActions({
-  [combineActions(increment, decrement)](state, { payload: { amount } }) {
```

Comment:
This is btw correct: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions